frontend/src/store/index.js

```diff
@@ -134,35 +134,47 @@ const state = {
   projectTerminalShortcuts: null,
   conditionCache: {
     APIServerAvailable: {
-      displayName: 'API Server',
+      name: 'API Server',
       shortName: 'API',
-      description: 'Indicates whether the shoot\'s kube-apiserver is healthy and available. If this is in error state then no interaction with the cluster is possible. The workload running on the cluster is most likely not affected.'
+      description: 'Indicates whether the shoot\'s kube-apiserver is healthy and available. If this is in error state then no interaction with the cluster is possible. The workload running on the cluster is most likely not affected.',
+      weight: '0'
```

Comment:
Why is weight a string and not a number?

Reply:
If you mix numbers and strings lodash `_.sortBy` method will not work properly. As shortname is the default this does not work. I'm not sure why it works in the orderBy code. Makes sense to discuss this briefly.

Reply:
I agree that mixin datatypes would be a bad idea. That's not what I had in mind. Ithough that the property name weight for me implies taht in all cases the values are numbers and not strings. For the known conditions this is obviously not a problem. For the unknown conditions the question is: What is the correct order?